monkey/infection_monkey/dropper.py

```diff
@@ -164,9 +165,10 @@ def start(self):
                 "monkey_commandline": inner_monkey_cmdline,
             }
 
+        monkey_cmdline_split = shlex.split(monkey_cmdline)
```

Comment:
Why do we need to split this up? Are you sure that without `shell=True` this works? Can the command still find environmental variables like `%TEMP%` or `$HOME`?